app/assets/stylesheets/content/_tabs.sass

```diff
@@ -69,12 +69,11 @@
   box-sizing: content-box
   height: 40px
   border-bottom: 1px solid #DDDDDD
+  margin-bottom: 1rem
   .tabrow
     display: block
     overflow-x: auto
     white-space: nowrap
-    // Padding for hiding the scroll bar
```

Comment:
@HDinger I guess this once was necessary, but when I tested it now without this style it also seemed to work fine. But we should have an eye on that...